docs/guides/js-waku/light-send-receive.md

```diff
@@ -24,6 +24,19 @@ await node.start();
 When the `defaultBootstrap` parameter is set to `true`, your node will be bootstrapped using the [default bootstrap method](/guides/js-waku/configure-discovery#default-bootstrap-method). Have a look at the [Bootstrap Nodes and Discover Peers](/guides/js-waku/configure-discovery) guide to learn more methods to bootstrap nodes.
 :::
 
+By default, your node will route messages to the standard pubsub topic (`/waku/2/default-waku/proto`). If your project uses a different shared pubsub topic, you can configure this using the `ShardInfo` parameter:
+
+```js
+// Create the shard info
+const shardInfo = { clusterId: 3, shards: [1, 2] };
+
+// Create node with custom shard info
+const node = await createLightNode({
+  defaultBootstrap: true,
+  shardInfo: shardInfo,
```

Comment:
Another way to `createLightNode` now would be to 
```ts
createLightNode({
defaultBootstra: true,
contentTopics: [/*set of topics*/],
});
```

Can we add it also to the docs, please?

Reply:
Nice, I'll update that too